core/chains/evm/logpoller/log_poller.go

```diff
@@ -497,21 +498,20 @@ func (lp *logPoller) run() {
 				}
 				// Otherwise this is the first poll _ever_ on a new chain.
 				// Only safe thing to do is to start at the first finalized block.
-				latest, err := lp.ec.HeadByNumber(lp.ctx, nil)
+				latestBlock, lastFinalizedBlockNumber, err := lp.latestBlocks(lp.ctx)
```

Comment:
nit: prefer latestFinalizedBlockNumber to lastFinalizedBlockNumber, more clear and aligned with latestBlock

Reply:
How about the database field? Do you have any suggestions? Should we keep it `last_finalized_block_number` or change to `latest`, or maybe only `finalized_block_number`? This is the last time we can easily change it ;)

Reply:
yeah I like just finalized_block_number